lib/client.js

```diff
@@ -3233,12 +3233,27 @@ MatrixClient.prototype.requestEmailToken = function(email, clientSecret,
     return this._http.idServerRequest(
         callback, "POST", "/validate/email/requestToken",
         params, httpApi.PREFIX_IDENTITY_V1
-    ).then(function(res) {
-        if (typeof res === "string") {
-            return JSON.parse(res);
-        }
-        return res;
-    });
+    );
+};
+
+/**
+ * Looks up the public Matrix ID mapping for a given 3rd party
+ * identifier from the Identity Server
+ * @param {string} medium The medium of the threepid, eg. 'email'
+ * @param {string} address The textual address of the threepid
+ * @param {module:client.callback} callback Optional.
+ * @return {module:client.Promise} Resolves: TODO
```

Comment:
aside: y'know, just because we've been rubbish at documenting these so far doesn't mean that we can't start...